API.md

```diff
@@ -459,6 +460,25 @@ Runs the given function with the supplied value, and then resolves with that val
 tapP(a => Promise.resolve('b'), 'a') //=> Promise 'a'
 ```
 
+### useWithP
+
+`@articulate/funky/lib/useWithP`
+
+```haskell
+useWithP :: (b -> c -> Promise d) -> [(a -> Promise b), (a -> Promise c)] -> a -> Promise d
```

Comment:
This looks shared with `convergeP`, which doesn't have the same signature.  `convergeP` takes a **single** arg, splits across multiple transformers, and then converges back to a single return value.  `useWithP` takes **multiple** args, transforms each one, and then applies them to the original function.  So maybe more like:
```haskell
useWithP :: (a -> b -> Promise c) -> [(d -> Promise a), (e -> Promise b)] -> (d -> e -> Promise c)
```

Reply:
Yeah, i copy/pasted this. I made an incorrect assumption since `converge(P)` returns a function that _can_ accept more than one argument. For converge, the converging function has an arity of 1 whereas the transforming functions can have any arity.